src/bolt/bolt_api.c

```diff
@@ -751,6 +758,7 @@ void BoltAcceptHandler
 	}
 
 	bolt_client_t *client = bolt_client_new(socket, global_ctx, BoltResponseHandler);
+	raxInsert(clients, (unsigned char *)&socket, sizeof(socket), client, NULL);
```

Comment:
Up to you, if you want to switch from `(unsigned char *)&socket, sizeof(socket)` to `(unsigned char *)&client, sizeof(client)`